src/components/ChatEntry.js

```diff
@@ -1,22 +1,28 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
 
 const ChatEntry = (props) => {
-  return (
+return(
     <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
+      <h2 className="entry-name">{props.sender}</h2>
       <section className="entry-bubble">
-        <p>Replace with body of ChatEntry</p>
-        <p className="entry-time">Replace with TimeStamp component</p>
-        <button className="like">🤍</button>
+        <p>{props.body}</p>
+        <p className="entry-time"><TimeStamp time={props.timeStamp}/></p>
+        <button onClick={() => props.toggleLike(props.id)} 
+        className="like">{props.isLiked ? '❤️' : '🤍' }</button>
       </section>
     </div>
   );
-};
-
+}
+      
 ChatEntry.propTypes = {
-  //Fill with correct proptypes
```

Comment:
Don'r forget you also want to include the `toggleLike` function you are passing in as a prop as well!